landd_intergration/README.md

```diff
@@ -0,0 +1,97 @@
+# OpenEdx Data Integration with L&D Services
+
+The Following Integration includes the transfer of course details and user's consumption data from OpenEdx to L&D services
+1. All the Meta Data of Courses created by staff in Openedx will be sent to L&D servies which shows up on Infopedia site
+2. User's entered in Infopedia courses will be redirected to OpenEdx site based on the Course URL
+3. Course Consumption data will be sent back to Infopedia SITE
+
+Here are the API's used: 
+
+**OpenEdx:**    
+
+  - Course catalog API         -- GET /api/courses/v1/courses/
+  - Bulk Gradees API           -- GET /api/grades/v1/user_grades/?username=all
+
+
+  
+**L&D servies:**
+
+  - Catalog_UpdateMultipleCatalogAsync                -- POST /Catalog/{sourceSystemId}/course
+  - Consumption_SaveMultipleExperienceTrackingAsync   -- POST /Consumption/exptrack
+
+  
+  
+
+**Logging**
+
+
+    All the logs related to course catalog are logged  in `course_catalog.log`
+	All the logs related to course consumption are logged in `course_consumption.log`
+	By default the LOG files will be stored in seperate files for every 24 hours
+	By default logs for the past 10 days are stored
+
+	
+	
+	
+**Time Logging**
+
+
+    
+    Bulk Gradees API acccepts start_data and end_date as parameters:
+	
+	start_data:For each Execution of script, time of the API call for Bulk Gradees API is logged in `api_call_time.txt` which will be used as start time for the subsequent run
+    end_date:By default end_date will be taken as current date and time
+
+		
+		
```

Comment:
nit: the above indentation is inconsistent (tabs and spaces). i recommend a) normalizing across these files and b) using an editor like vs code which will help enforce consistency